### PR TITLE
Standardize list output format and coloring

### PR DESCRIPTION
**Pull request by ianlivingstone**
A few of our list commands did not output the number of items in the
list and others did not properly colorize the integer.

I've gone through and standardized these across Torus.

### CODE REVIEW

File: cmd/orgs.go [hunk 1]
```diff
@@ -145,7 +146,8 @@ func orgsListCmd(ctx *cli.Context) error {
 		fmt.Printf("%s\n", o.Body.Name)
 	}
 
-	hints.Display(hints.PersonalOrg)
+	fmt.Printf("\nYou are a member of (%s) org%s\n",
```
Comment (MattAWright) [inline cmd/orgs.go]: No more personal org hint?

File: cmd/signup.go [hunk 2]
```diff
@@ -80,7 +80,7 @@ func signup(ctx *cli.Context, subCommand bool) error {
 	}
 
 	reminderLabel := "Reminder: "
-	ui.Hint("Don't forget to keep your password safe and secure! You can't recover your account if you lossswordd.", false, &reminderLabel)
```
Comment (MattAWright) [inline cmd/signup.go]: How did that even get in there?

Comment (ianlivingstone) [reply cmd/signup.go]: The magic of random key mashing in vim

bitmoji

File: cmd/orgs.go (hunk 1 above)
Comment (ianlivingstone) [reply cmd/orgs.go]: I removed it because it made the output really messy, would be fine with adding it back.

**Pull request merged**